src/styles/main.scss

```diff
@@ -1,3 +1,77 @@
+$width-day: 100px;
+$gap: 1px;
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+}
+
+html {
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+}
+
+@mixin square ($width-day) {
+  width: $width-day;
+  height: $width-day;
+}
+
+@for $index from 1 through 31 {
+  .calendar__day:nth-child(#{$index})::before {
+    content: "#{$index}";
+    line-height: $width-day;
+  }
+}
+
+$first-day:
+  "mon" 0,
+  "tue" 1,
+  "wed" 2,
+  "thu" 3,
+  "fri" 4,
+  "sat" 5,
+  "sun" 6;
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  max-width: 726px;
```

Comment:
Would suggest creating a variable for max-width, but it's totally up to you. Overall, the code is good and clean!:)